### PR TITLE
Fix/protocal modal style

### PR DESCRIPTION
**Pull request by GrigoriGrigoryanBytepitch**

### CODE REVIEW

File: src/components/AddProtocolModal/AddProtocolModal.jsx
Comment (anilhelvaci) [inline src/components/AddProtocolModal/AddProtocolModal.jsx:1]: This component doesn't seem to be have the js code in `Alternative.jsx` @GrigoriGrigoryanBytepitch

File: src/components/AddProtocolModal/Alternative.jsx [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    };
+
+    console.log('utilityPurse: ', utilityPurse);
+
+    return (
+        open && (
+            <ModalWrapper className="add-protocol-modal">
+                {/*{submittedData && <RentalCreator data={submittedData} onSubmit={() => setSubmittedData(null)} />}*/}
+                <header>
+                    <h2 className="modal-title">Add Crabble Protocol</h2>
```
Comment (anilhelvaci) [inline src/components/AddProtocolModal/Alternative.jsx]: Nor this component has the styling of `AddProtocolModal.jsx` @GrigoriGrigoryanBytepitch

Comment (GrigoriGrigoryanBytepitch) [reply src/components/AddProtocolModal/Alternative.jsx]: I did not add style for header, but now pushed. Can you please check it again ?

Comment (anilhelvaci) [reply src/components/AddProtocolModal/Alternative.jsx]: Looks great man! I did some cleanup, merging now.

**Pull request merged**